packages/qwik/src/optimizer/src/plugins/image-size-server.ts

```diff
@@ -107,33 +109,34 @@ export async function getInfoForSrc(src: string) {
 export const getImageSizeServer = (
   sys: OptimizerSystem,
   rootDir: string,
-  srcDir: string
+  srcDir: string,
+  server: ViteDevServer
 ): Connect.NextHandleFunction => {
   return async (req, res, next) => {
-    const fs: typeof import('fs') = await sys.dynamicImport('node:fs');
-    const path: typeof import('path') = await sys.dynamicImport('node:path');
+    try {
```

Comment:
So is the fix that errors are caught now? Then the changelog could be "be more robust about errors in the qwik city image info service"